pkg/providers/aws/cluster_vpc.go

```diff
@@ -457,37 +457,53 @@ func getSubnets(ec2Svc ec2iface.EC2API) ([]*ec2.Subnet, error) {
 	return subs, nil
 }
 
+func getVPCIDByClusterSubnets(ec2Svc ec2iface.EC2API, clusterID string) (*string, error) {
```

Comment:
Do we need to return a pointer to a string here? I might clean it up a little bit to avoid returning `aws.String("")`

Reply:
addressed. Now returning `nil` instead of a pointer to an empty string

Reply:
Is there a reason why this is being returned at a pointer?

Reply:
addressed. Pointers in no more